trunk/includes/pardot-settings-class.php

```diff
@@ -932,212 +863,118 @@ function settings_page() {
 	/**
 	 * Returns HTML input name for a raw field name
 	 *
-	 * The Settings API want HTML input names in the form "pardot_settings[email]" instead of just "email".
+	 * The Settings API want HTML input names in the form "pardot_settings[client_id]" instead of just "client_id".
 	 *
 	 * @param string $field_name
 	 * @return string
 	 *
 	 * @since 1.0.0
 	 */
-	private function _get_html_name( $field_name ) {
-		return self::$OPTION_GROUP  . "[{$field_name}]";
+	private function _get_html_name($field_name)
+	{
+		return self::$OPTION_GROUP . "[{$field_name}]";
 	}
 
-    /**
-     * Displays authentication status
-     *
-     * @since 1.5.0
-     */
-	function auth_status_field() {
-	    if (self::is_authenticated() && self::get_api(array())->get_account()) {
-	        if ( self::get_setting('auth_type') == 'sso' ) {
-                $message = __('Authenticated with Salesforce SSO', 'pardot');
-                $buttonValue = __( 'Re-authenticate with Salesforce', 'pardot' );
-                $html =<<<HTML
-<div id="auth-status-wrap" class="success">
-{$message}
-<input id="sso-sign-in" class="button-primary" name="sso-sign-in" style="width: 217px" value="{$buttonValue}" onclick="clickSubmit()"/>
-</div>
-HTML;
-            }
-	        else {
-                $message = __('Authenticated with Pardot', 'pardot');
-                $html =<<<HTML
+	/**
+	 * Displays authentication status
+	 *
+	 * @since 1.5.0
+	 */
+	function auth_status_field()
+	{
+		if (self::is_authenticated() && self::get_api([])->get_account()) {
+			$message = __('Authenticated with Salesforce SSO', 'pardot');
+			$buttonValue = __('Re-authenticate with Salesforce', 'pardot');
+			$html = <<<HTML
 <div id="auth-status-wrap" class="success">
 {$message}
+<input type="button" id="sso-sign-in" class="button-primary" name="sso-sign-in" style="width: 217px" value="{$buttonValue}" onclick="clickSubmit()"/>
```

Comment:
This prevents the Authenticate button from acting like an input text field.  The cursor could be placed in the button text, which certainly isn't right.